### PR TITLE
Add details of training to json output for benchmark

### PR DESCRIPTION
**Pull request by hadusam**
## Motivation and Context

Add details of training to `lmnet/executor/evaluate.py` for the benchmark.

## How has this been tested?

The example of JSON as follows.
```
{
    "task_type": "IMAGE.OBJECT_DETECTION",
    "network_name": "LMFYolo",
    "dataset_name": "coco2014",
    "dataset_path": "gs://lmfs-backend-us-west1/shared/tensorflow_datasets",
    "last_step": 17000,
    "metrics": [
        {
            "name": "MeanAveragePrecision_0.3",
            "value": 0.005068858583270216
        },
        {
            "name": "AveragePrecision_0.5/person",
            "value": 0.08731818928337996
        },
        {
            "name": "Recall_0.5/person",
            "value": 0.28203441352948516
        },
...
    ]
}
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: lmnet/executor/evaluate.py [hunk 1]
```diff
@@ -150,7 +150,15 @@ def evaluate(config, restore_path, output_dir):
     )
     validation_writer.add_summary(metrics_summary, last_step)
 
+    is_tfds = "tfds_kwargs" in config.DATASET
+    dataset_name = config.DATASET.tfds_kwargs["name"] if is_tfds else config.DATASET_CLASS.__name__
+    dataset_path = config.DATASET.tfds_kwargs["data_dir"] if is_tfds else ""
+
     metrics_dict = {
+        'task_type': config.TASK.value,
+        'model_name': config.NETWORK_CLASS.__name__,
```
Comment (iizukak) [inline lmnet/executor/evaluate.py]: In Blueoil, we often use `model_name` as config name.
https://github.com/blue-oil/blueoil/blob/master/blueoil/generate_lmnet_config.py#L120

I think It's better to use `network_name`.  How about?

Comment (hadusam) [reply lmnet/executor/evaluate.py]: @iizukak Thanks! I think it is better too.
Changed 👍

**Comment by hadusam** [comment]
@iizukak
I added an example of JSON output, please re-review! 🙇

**Comment by hadusam** [comment]
@tkng 
Please review readability for Python 👍

**Pull request merged**